examples/local_live_view/lib/local_live_view/js.ex

```diff
@@ -0,0 +1,18 @@
+defmodule LocalLiveView.JS do
```

Comment:
Either add moduledoc or `@moduledoc false`. Generally, generate documentation and see how it looks like